src/Admin/Startup.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.AspNetCore.Mvc.Razor;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Bit.Admin.Services;
+using Bit.Commercial.Infrastructure.EntityFramework.SecretsManager;
```

Comment:
This needs to be moved within the `#if !OSS` block (~line 15 below). That ensures that it's only included in the compilation for non-OSS builds.

Reply:
done